dead.py

```diff
@@ -55,12 +55,15 @@ def visit_ImportFrom(self, node: ast.ImportFrom) -> None:
 
     def visit_ClassDef(self, node: ast.ClassDef) -> None:
         if not self.is_test:
-            self.defines[node.name].add(self.filename)
+            self.defines[node.name].add(self.definition_str(node))
         self.generic_visit(node)
 
+    def definition_str(self, node: ast.AST) -> str:
+        return f"{self.filename}:{node.lineno}"
```

Comment:
can you add `double-quote-string-fixer` to the pre-commit config? it comes from `pre-commit-hooks`

Reply:
How about adding this `pre-commit-config` to a cookie-cutter template? 😸

Reply:
heh I mean tbh I didn't expect anyone to contribute to this and if I follow it already... 😆